plugins/inputs/couchbase/README.md

```diff
@@ -22,7 +22,7 @@
 ### couchbase_node
 
 Tags:
-- cluster: whatever you called it in `servers` in the configuration, e.g.: `http://couchbase-0.example.com/`
+- cluster: sanitized and scheme less string from `servers` configuration field e.g.: `http://user:password@couchbase-0.example.com:8091/endpoint` -> `couchbase-0.example.com:8091/endpoint`
```

Comment:
I agree about the concern on multiple servers differing by scheme only, but sympathize with respect to your style concerns.   That said, we should keep the scheme if only to preserve backwards compatibility.